exporter/awscloudwatchlogsexporter/README.md

```diff
@@ -13,25 +13,23 @@
 
 AWS CloudWatch Logs Exporter sends logs data to AWS [CloudWatch Logs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html).
 AWS credentials are retrieved from the [default credential chain](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
-Region must be configured in the configuration if not set in the default credential chain.
```

Comment:
this is not true anymore?

Reply:
No it is, I just moved this information to the configuration section. Thought it would be more clear to put the information there.